options.js

```diff
@@ -1,32 +1,35 @@
 var tcDefaults = {
-  speed: 1.0,           // default 1x
-  speedStep: 0.1,       // default 0.1x
-  rewindTime: 10,       // default 10s
-  advanceTime: 10,      // default 10s
-  fastSpeed: 1.8,       // default 1.8x
-  resetKeyCode:  82,    // default: R
-  slowerKeyCode: 83,    // default: S
-  fasterKeyCode: 68,    // default: D
-  fastKeyCode: 71,      // default: G
-  rewindKeyCode: 90,    // default: Z
-  advanceKeyCode: 88,   // default: X
+  speed: 1.0,           // default:
   displayKeyCode: 86,   // default: V
   rememberSpeed: false, // default: false
   startHidden: false,   // default: false
+  keyBindings: [
+    ["slower", 83, 0.1, 0], // default: S 0.1
```

Comment:
To improve readability / ergonomics, let's use an object instead? 

`
[ 
  { "action": "slower", "keycode": ...}, 
  ...
] 
`